packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -241,6 +235,186 @@ PageHeaderContent.propTypes = {
   pageActions: PropTypes.node,
 };
 
+/**
+ * ----------------
+ * PageHeaderContentPageActions
+ * ----------------
+ */
+interface PageHeaderContentPageActionsProps {
+  /**
+   * Provide child elements to be rendered inside PageHeaderContentPageActions.
+   */
+  children?: React.ReactNode;
+  /**
+   * Specify an optional className to be added to your PageHeaderContentPageActions
+   */
+  className?: string;
+  /**
+   *  The PageHeaderContent's page actions collabpsible Menu button label
+   */
+  menuButtonLabel?: string;
+  /**
+   * The PageHeaderContent's page actions
+   */
+  pageActions?: React.ReactNode;
+}
+const PageHeaderContentPageActions = React.forwardRef<
+  HTMLDivElement,
+  PageHeaderContentPageActionsProps
+>(function PageHeaderContentPageActions(
+  {
+    className,
+    children,
+    menuButtonLabel = 'Actions',
+    pageActions,
+    ...other
+  }: PageHeaderContentPageActionsProps,
+  ref
+) {
```

Comment:
Should this function be simplified to arrow function which is what most other components use? I have this question for other components in this file as well because function names are being duplicated which doesn't seem necessary.

```suggestion
>((
  {
    className,
    children,
    menuButtonLabel = 'Actions',
    pageActions,
    ...other
  }: PageHeaderContentPageActionsProps,
  ref
) => {
```